app/Actions/AfterCommands.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace App\Actions;
+
+use App\Support\BaseAction;
+
+class AfterCommands extends BaseAction
+{
+    /**
+     * Perform the After Commands in Project Directory.
+     *
+     * @return void
+     */
+    public function __invoke(): void
+    {
+        $directory  = config('lambo-store.project_path');
```

Comment:
How hard would it be to move these config into a `lambo` directory? I assume we'd then be using `lambo.store.project_path`

Reply:
I think so yah